src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
+
+  if (days < 3) {
+    return result;
+  } else if (days >= 3 && days < 7) {
```

Comment:
4. [Checklist] Prefer `if` with return over `if else` to simplify later conditions.